proto/storage/billing_metrics.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "storage";
+
+message BillingMetrics {
+    message SecuredResources {
+        int32 nodes = 1;
+        int32 cores = 2;
+    }
+
+    google.protobuf.Timestamp ts = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    SecuredResources sr = 2;
```

Comment:
I'd flatten that out and call the fields `num_secured_nodes` and `num_secured_vcpus` for simplicity.

Reply:
can also called it secured_resources if want to keep the nested structure